src/VolleyManagement.Backend/VolleyManagement.Domain/PlayersAggregate/Player.cs

```diff
@@ -14,6 +14,66 @@ public class Player
         private short? _height;
         private short? _weight;
 
+        /// <summary>
+        /// Initializes a new instance of the Player
+        /// </summary>
+        /// <param name="id">Id</param>
+        /// <param name="firstName">Fisrt Name</param>
+        /// <param name="lastName">Last Name</param>
+        public Player(int id, string firstName, string lastName) : this(id, firstName, lastName, null, null, null)
+        {
+
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the Player
+        /// </summary>
+        /// <param name="id">Id</param>
+        /// <param name="firstName">First Name</param>
+        /// <param name="lastName">Last Name</param>
+        /// <param name="birthYear">BirthYear</param>
+        /// <param name="height">Height</param>
+        /// <param name="weight">Weight</param>
+        public Player(int id, string firstName, string lastName, short? birthYear, short? height, short? weight)
+            : this(id, firstName, lastName, birthYear, height, weight, null)
+        {
+
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the Player
+        /// </summary>
+        /// <param name="firstName">First Name</param>
+        /// <param name="lastName">Last Name</param>
+        /// <param name="birthYear">BirthYear</param>
+        /// <param name="height">Height</param>
+        /// <param name="weight">Weight</param>
+        public Player(string firstName, string lastName, short? birthYear, short? height, short? weight, int? teamId)
+            : this(0, firstName, lastName, birthYear, height, weight, teamId)
+        {
+
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the Player
+        /// </summary>
+        /// <param name="id">Id</param>
+        /// <param name="firstName">First Name</param>
+        /// <param name="lastName">Last Name</param>
+        /// <param name="birthYear">BirthYear</param>
+        /// <param name="height">Height</param>
+        /// <param name="weight">Weight</param>
+        public Player(int id, string firstName, string lastName, short? birthYear, short? height, short? weight, int? teamId)
```

Comment:
This constructor should have entity invariant check to make sure instance created is correct